services/user/FractalSys/ui/vite.config.ts

```diff
@@ -52,11 +52,11 @@ const psibase = (options: Options) => {
             ...httpServerConfig,
             https: {
                 key: fs.readFileSync(
-                    "./psibase.127.0.0.1.sslip.io+1-key.pem",
+                    "../../../../psibase.127.0.0.1.sslip.io+1-key.pem",
```

Comment:
Can the vite config read environment variables? Might be nice to set the path to the location of the cert and key as an environment variable and not have to locally touch the vite config.

Reply:
Yeah, good call.